### PR TITLE
feat: HCMP-8 migrate THC

### PR DESCRIPTION
**Pull request by mrmartineau**
# What?

HCMP-8 migrate thc to this project

Minor changes have been made to get the app running but not much else.

# Why?

Home Check is extremely similar to THC so we copied it.

### CODE REVIEW

File: __tests__/features/end-to-end.feature [hunk 1]
```diff
@@ -1,8 +1,8 @@
 Feature: End to end
 
   As a user
-  I want to be able to perform a Home Check
-  So that I can check a home
+  I want to be able to perform a Tenancy and Household Check
+  So that I can check on a tenancy
```
Comment (LBHABEBIC) [inline __tests__/features/end-to-end.feature:6]: is this right?

File: __tests__/pages/hc/index.test.tsx [hunk 2]
```diff
@@ -35,8 +35,8 @@ it("redirects to the loading page when online", async () => {
 
   expect(routerReplaceMock).toHaveBeenCalledTimes(1);
   expect(routerReplaceMock).toHaveBeenCalledWith(
-    { pathname: "/hc/[processRef]/loading" },
-    { pathname: "/hc/test-process-ref/loading" }
+    { pathname: "/thc/[processRef]/loading" },
+    { pathname: "/thc/test-process-ref/loading" }
   );
 });
```
Comment (LBHABEBIC) [inline __tests__/pages/hc/index.test.tsx:42]: should this be thc?

**Review by LBHABEBIC — APPROVED** [review]
looks good!

File: __tests__/pages/hc/index.test.tsx (hunk 2 above)
Comment (mrmartineau) [reply __tests__/pages/hc/index.test.tsx:42]: those will be changed in one of the subsequent tickets

File: __tests__/features/end-to-end.feature (hunk 1 above)
Comment (mrmartineau) [reply __tests__/features/end-to-end.feature:6]: that will be changed in one of the subsequent tickets

**Pull request merged**